README.md

```diff
@@ -58,29 +64,45 @@ The following tutorial applies to Linux / MacOS development environments. Visit
 
 type: "cd .." to go back to the toynet-flask directory
```

Comment:
We could break this out into some descriptive text and then the `cd` command in a code block, similar to the others. As it is here, it might be easy to miss.

Reply:
I'm going to include the python download link on line 8 as well but do you think I should include the teminal instructions here?

Man, if only I was stacking this money instead of repeatedly pressing `shift`+`4`

Reply:
Download link is a good idea; terminal instructions will vary by OS for the installation. If you want, you can write the instructions for how to install Python for MacOS and Debian-flavored Linux distros, but that's not super important since most people will follow the instructions at the link.